doc/tutorials/starship.rst

```diff
@@ -89,12 +102,94 @@ To execute this file using Python, you can use a code block like this:
    result = eng.run(prog)
    print(result.samples)
 
-Executing your xbb file from the command line
-=============================================
+
+Executing your Blackbird script from the command line
+-----------------------------------------------------
+
 To execute this file from the command line, use the ``starship`` command as follows:
 
 .. code-block:: console
 
    starship --input test.xbb --output out.txt
 
-After executing the above command, the result will be stored in ``out.txt`` in the current working directory. You can also omit the ``--output`` parameter to print the result to the screen.
+After executing the above command, the result will be stored in ``out.txt`` in the current working directory.
+You can also omit the ``--output`` parameter to print the result to the screen.
+
+
+Program compilation
+===================
+
+In addition to using the program template above, which directly matches the physical
+layout of the hardware device, you can define your interferometer using any combination
```

Comment:
Maybe: "..., you can apply any two-mode interferometer to the pairs of modes. The interferometer can be composed of any combination ..."

Reply:
```suggestion
layout of the hardware device, you can apply any two-mode interferometer to the pairs of modes.
The interferometer can be composed of any combination
```